bindings/java/c/host.h

```diff
@@ -0,0 +1,21 @@
+#include "evmc/evmc.h"
+#include <jni.h>
+
+#ifndef _Included_org_ethereum_evmc_Host
+#define _Included_org_ethereum_evmc_Host
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+JavaVM* jvm;
```

Comment:
Is this variable actually defined in the header?

Reply:
it is used in host.c and evmc-vm.c.  i could probably make some kind of a helper method and eliminate the need for it in host.h

Reply:
@chfast any preference?

Reply:
```suggestion
extern JavaVM* jvm;
```
This should have been a declaration as a variable (it is a definition). Then you can define it as `JavaVM* jvm` in host.c.

A slightly better way is to make it `static JavaVM* jvm` in host.c and add helper `set_jvm(JavaVM* jvm)` to allow set it from evmc-vm.c. This is still a global, but at least we keep it only in single file.

Reply:
I will update with the latter suggestion

Reply:
updating with the 'preferred' method

Reply:
Need to actually declare `JavaVM *jvm;` in `host.c`. The change here means that `jvm` will only created once and compiler doesn't need to resolve which file owns it. (It is basically whichever file was compiled first in the pipeline including `host.h.`

Reply:
I understand how externs work...it was a mistake.  i didn't really want that change - I fixed it with a helper method.